jobs/integr8ly/ocp4/nightly/trigger/aws/osd-ocp4-master-aws-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,34 @@
+---
+
+- job:
+    name: osd-ocp4-master-aws-testing-nightly-trigger
+    display-name: OpenShift 4 OSD Master AWS Nightly Testing Trigger
+    project-type: pipeline
+    description: "Nightly trigger for osd ocp4 testing using master branch and AWS resources"
+    sandbox: false
+    concurrent: false
+    triggers:
+      - timed: "H(0-10) H(3-4) * * *"
```

Comment:
can you please change the hour to H(4-5) so it tis not starting at the same time as master installation? Otherwise I think it is LGTM.